cdc/generation.cc

```diff
@@ -154,8 +154,9 @@ topology_description generate_topology_description(
     tokens.erase(std::unique(tokens.begin(), tokens.end()), tokens.end());
 
     std::vector<token_range_description> entries(tokens.size());
-    int spots_to_fill = 0;
 
+    assert(tokens.size() > 1);
```

Comment:
Why > 1? What if we have a single node with one token?

Reply:
This assumption just simplified the code. I thought that we always use vnodes but if you want to cover the case of a node having just one token then I will fix the code.

Reply:
I don't think you depend on it.

And yes, we always use vnodes, but we don't want to add secret dependencies on vnodes.

Reply:
Sure. Will fix.

Reply:
done.